lib/vsc/install/shared_setup.py

```diff
@@ -166,13 +175,56 @@ def find_package_modules (self, package, package_dir):
 
 
 class vsc_bdist_rpm(bdist_rpm):
-    """ Custom class to build the RPM, since the __inti__.py cannot be included for the packages that have namespace spread across all of the machine."""
+    """Custom class to build the RPM, since the __init__.py cannot be included for the packages that have namespace spread across all of the machine."""
     def run(self):
         log.error("vsc_bdist_rpm = %s" % (self.__dict__))
         SHARED_TARGET['cmdclass']['egg_info'] = vsc_bdist_rpm_egg_info  # changed to allow removal of files
         self.run_command('egg_info')  # ensure distro name is up-to-date
         orig_bdist_rpm.run(self)
 
+class VscTestCommand(TestCommand):
+    """
+    The cmdclass for testing
+    """
+    def run_tests(self):
+        # should be setup.py
+        setup_py = inspect.stack()[-1][1]
+        log.info('run_tests from %s' % setup_py)
```

Comment:
i pulled the inspect code form stackoverflow, rereading it, they tried to solve something more complicated, so i'll revert to `sys.argv[0]`